packages/mgt-components/src/graph/graph.photos.ts

```diff
@@ -120,6 +120,8 @@ export const getUserPhoto = async (graph: IGraph, userId: string): Promise<strin
   let cache: CacheStore<CachePhoto>;
   let photoDetails: CachePhoto;
 
+  const encodedUser = `${userId.replace(/#/g, '%2523')}`;
```

Comment:
Is the `encodeIURIComponent` api a better _encoder_ here? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent

Reply:
Also, `#` gives `%23`. Where is `%2523` coming from?

Reply:
Using just # encoding was resulting into the same issue, while using double encoding, `2523` fixed it.